tests/test-sources/plugins/telescope/default.nix

```diff
@@ -16,6 +16,7 @@
       };
       keymapsSilent = true;
       highlightTheme = "gruvbox";
+      enabledExtensions = [ "some_extension" ];
```

Comment:
Can't test this because the extension won't exist...

```
E5113: Error while calling lua chunk: .../start/telescope.nvim/lua/telescope/_extensions/init.lua:10: 'some_extension' extension doesn't exist or isn't installed
```